operator/src/lib.rs

```diff
@@ -94,101 +163,103 @@ impl KafkaState {
-                    let cm = stackable_operator::create_config_map(
-                        &self.context.resource,
-                        &tmp_name,
-                        data,
-                    )?;
-                    self.context.client.create(&cm).await?;
-                    pod
-                }
-                Some(pod) => {
-                    trace!(
-                        "Found existing pod [{}] for broker [{}]",
-                        Meta::name(pod),
-                        broker.node_name
-                    );
-                    pod.clone()
-                }
-            };
-
-            // If the pod for this server is currently terminating (this could be for restarts or
-            // upgrades) wait until it's done terminating.
-            if finalizer::has_deletion_stamp(&pod) {
-                info!("Waiting for Pod [{}] to terminate", Meta::name(&pod));
-                return Ok(ReconcileFunctionAction::Requeue(Duration::from_secs(10)));
-            }
+        let config = handlebars
+            .render("conf", &json!({ "options": options }))
+            .unwrap();
 
-            // At the moment we'll wait for all pods to be available and ready before we might enact any changes to existing ones.
-            // TODO: Only do this next check if we want "rolling" functionality
-            if !podutils::is_pod_running_and_ready(&pod) {
-                info!(
-                    "Waiting for Pod [{}] to be running and ready",
-                    Meta::name(&pod)
-                );
-                return Ok(ReconcileFunctionAction::Requeue(Duration::from_secs(10)));
-            }
-        }
+        let mut data = BTreeMap::new();
+        data.insert("server.properties".to_string(), config);
 
-        Ok(ReconcileFunctionAction::Continue)
+        // And now create the actual ConfigMap
+        let cm =
+            stackable_operator::config_map::create_config_map(&self.context.resource, &name, data)?;
+        self.context.client.create(&cm).await?;
+        Ok(())
     }
 
-    pub async fn delete_excess_pods(&self) -> KafkaReconcileResult {
-        trace!("Starting to delete excess pods",);
-
-        // We iterate over all Pods that have an OwnerReference pointing at us
-        // and we'll try to find all pods that are _not_ assigned to any
-        // broker from the spec, we then delete it.
-        for pod in &self.existing_pods {
-            if !self
-                .kafka_spec
-                .brokers
-                .iter()
-                .any(|broker| podutils::is_pod_assigned_to_node(pod, &broker.node_name))
-            {
-                self.context.client.delete(pod).await?;
-                return Ok(ReconcileFunctionAction::Requeue(Duration::from_secs(10)));
+    async fn create_missing_pods(&mut self) -> KafkaReconcileResult {
+        // The iteration happens in two stages here, to accomodate the way our operators think
```

Comment:
accommodate

Reply:
Typo still here: accomodate -> accommodate
// The iteration happens in two stages here, to accommodate the way our operators think